src/apps/explorer/components/SummaryCardsWidget/VolumeChart/VolumeChart.tsx

```diff
@@ -69,13 +70,8 @@ function _formatAmount(amount: string): string {
  *  example: <lastRecordId>-<volumePeriodSelected>
  * */
 function usePreviousLastValueData<T>(value: T): T | undefined {
-  const ref = useRef<T>()
-
-  useEffect(() => {
-    ref.current = value
-  }, [value])
-
-  return ref.current
+  const previousValue = usePrevious(value)
+  return previousValue
```

Comment:
Nit pick: What about returning the hook?

```suggestion
  return usePrevious(value)
```

Reply:
Actually, unless there's anything special about the types, this hook is not needed and you can use `usePrevious` instead